submarine-server/server-core/src/main/java/org/apache/submarine/server/SubmitterManager.java

```diff
@@ -52,7 +52,7 @@
     LOG.info("Submarine Home: {}", home);
     SUBMITTER_CONFIG_MAP.put("k8s", new JobSubmitterConfig(
         "org.apache.submarine.server.submitter.k8s.K8sSubmitter",
-        home + "/lib/submitter/k8s/"));
+        "/Users/byronhsu/Desktop/submarine/submarine-server/server-submitter/submitter-k8s/target"));
```

Comment:
let's not hardcode the path?

Reply:
@jojochuang  Thanks! It is a temporary commit, and I accidentally pushed it. There are still bugs in this PR. I am working on fixing it right now.